Source/WebInspectorUI/UserInterface/Controllers/NetworkManager.js

```diff
@@ -229,6 +230,14 @@ WI.NetworkManager = class NetworkManager extends WI.Object
     get mainFrame() { return this._mainFrame; }
     get localResourceOverrides() { return this._localResourceOverrides; }
     get bootstrapScript() { return this._bootstrapScript; }
+    
+    get failedSourceMapURLs() {
```

Comment:
Style: since these are simple `get` (i.e. they return a member variable of the same name) you can inline them at the end of the inlined `get` above
```
get bootstrapScript() { return this._bootstrapScript; }
get failedSourceMapURLs() { return this._failedSourceMapURLs; }
get downloadingSourceMaps() { return this._downloadingSourceMaps; }
```

Reply:
Addressed in e2c3874b5b35844fbcd028d68867b71662ae988e.